spec.html

```diff
@@ -36380,7 +36380,7 @@
     <!-- es6num="B.3.1" -->
     <emu-annex id="sec-__proto__-property-names-in-object-initializers">
       <h1>__proto__ Property Names in Object Initializers</h1>
-      <p>The following Early Error rule is added to those in <emu-xref href="#sec-object-initializer-static-semantics-early-errors"></emu-xref>:</p>
+      <p>The following Early Error rule is added to those in <emu-xref href="#sec-object-initializer-static-semantics-early-errors"></emu-xref>. When |ObjectLiteral| appears in a context where |ObjectAssignmentPattern| is required the Early Error rule is <b>not</b> applied. In addition, it is not applied when initially parsing a |CoverParenthesizedExpressionAndArrowParameterList|.</p>
```

Comment:
The text, including its use of `<b>`, was copied from [12.2.6.1](http://tc39.github.io/ecma262/#sec-object-initializer-static-semantics-early-errors), so if we decide to change the markup we probably want to change it here and in 12.2.6.1. Apart from that, [&lt;strong&gt;](https://html.spec.whatwg.org/multipage/semantics.html#the-strong-element) seems to be preferred over [&lt;em&gt;](https://html.spec.whatwg.org/multipage/semantics.html#the-em-element) in this context? (If that's not the case, just ignore my comment; the subtleties of semantic html are probably not my strong suit. :wink: )